README.md

```diff
@@ -43,6 +43,13 @@ Additionally the Bolt project includes:
     * [Writing Plans](https://puppet.com/docs/bolt/latest/writing_plans.html)
 * [Applying Manifest Blocks in Plans](https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html)
 
+## A Note on Kerberos
+
+We support using [Kerberos]() for authentication, but due to license incompatibility with other components we are distributing you'll need to install the gem yourself. You can do this with
```

Comment:
I don't think I'd include this in the README at all. I think it should go in the install docs as a note at the end.